DigitalLearningSolutions.Web/Views/NotificationPreferences/UpdateNotificationPreferences.cshtml

```diff
@@ -0,0 +1,19 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.MyAccount
+
+@model UpdateNotificationPreferencesViewModel
+
+<link rel="stylesheet" href="@Url.Content("~/css/myAccount/notificationPreferences.css")" asp-append-version="true">
+
+@{
+  ViewData["Title"] = "Update Notification Preferences";
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    <h1 class="nhsuk-heading-xl">Update Notification Preferences</h1>

```

Comment:
I know it's slightly different from the wireframes, but could we make this "Update notification preferences" (re capitalisation)